custom-recipes/to-excel/recipe.json

```diff
@@ -41,7 +41,7 @@
         {
             "name": "export_conditional_formatting",
             "label": "Apply conditional formatting",
-            "description": "Color cells by rules, when applicable",
+            "description": "Color cells by rules, when applicable (Only available for DSS 12.5+)",
```

Comment:
```suggestion
            "description": "Color cells by rules, when applicable (Only available for DSS 12.6+)",
```

Reply:
Done in 9b07c0fe7543b8f6e6bbe17acfdb5951ac2a7686